scripts/casefold_db.py

```diff
@@ -38,18 +80,72 @@ def calculate_lookup_hash(sydent, address):
     return lookup_hash
 
 
+def sendEmailWithBackoff(
+    sydent: Sydent,
+    address: str,
+    mxid: str,
+    backoff: int,
```

Comment:
Since we take `test` as an arg and backoff is always `0 if test else 1` then we could just get rid of this arg entirely and hard code it at the start :shrug: